packages/core/src/payment/payment-method-request-sender.ts

```diff
@@ -9,7 +9,7 @@ export default class PaymentMethodRequestSender {
         private _requestSender: RequestSender
     ) {}
 
-    loadPaymentMethods({ timeout }: RequestOptions = {}): Promise<Response<PaymentMethod[]>> {
+    loadPaymentMethods(cartId: string, { timeout }: RequestOptions = {}): Promise<Response<PaymentMethod[]>> {
```

Comment:
Think we can put `cartId` at the end as an optional param. As a result, no changes to the previous code are required.